validation/src/test/scala/hmda/validation/rules/lar/macro/Q008Spec.scala

```diff
@@ -0,0 +1,24 @@
+package hmda.validation.rules.lar.`macro`
+import akka.stream.scaladsl.Source
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.rules.AggregateEditCheck
+import hmda.validation.rules.lar.`macro`.MacroEditTypes.LoanApplicationRegisterSource
+
+class Q008Spec extends MacroSpec {
+
+  "Q008" must {
+    "be valid if withdrawn <= 0.30 * total" in {
+      larSource.mustPass
+    }
+
+    "be invalid if withdrawn > 0.30 * total" in {
+      val badLar1 = lars.head.copy(actionTakenType = 4)
+      val badLar2 = lars.head.copy(actionTakenType = 4)
```

Comment:
Since all lines are the same I think it makes a bit more sense to remove `badLar2` and add `badLar1` on multiple times.